bitcoin/src/blockdata/weight.rs

```diff
@@ -205,6 +219,15 @@ mod tests {
         let result = Weight(2).checked_div(0);
         assert_eq!(None, result);
     }
+
+    #[test]
+    fn scale_by_witness_factor_test() {
```

Comment:
According to Rust convention "test" shouldn't appear in unit test names (I know we violate this a bunch but lets not introduce more :)

```suggestion
    fn scale_by_witness_factor() {
```